src/components/BaseCanvas/BaseCanvas.component.tsx

```diff
@@ -17,7 +17,7 @@ export const BaseCanvas: FC<BaseCanvasProps> = ({
   fov = 50,
   position = new Vector3(0, 0, 5),
   style,
-  dpr = [1, 2],
+  dpr = [window.devicePixelRatio * 0.5, 2],
```

Comment:
playing with min devicePixelRatio improves and adapts the visage quality depending on the monitor we're running, changing the max does something strange on monitor 144Hz quality there is low compared with retina screen